proposal/color-4-new-spaces.md

```diff
@@ -1702,12 +1790,17 @@ This function is also available as a global function named `adjust-color()`.
 
   * Set `channel` to `channel + adjust`.
 
+    > This will throw an error if `adjust` and `channel` are not compatible.
```

Comment:
If `channel` is just the numeric value of the channel, then I don't think this addition will work automatically if `adjust` is a percentage, unless it's being converted to a plain value somewhere else I haven't noticed.